vite.config.js

```diff
@@ -77,7 +77,10 @@ export default ({ mode }) => {
         presets: [presetWarp()],
         mode: 'shadow-dom',
         safelist: classes,
-      }),  
+      }),
+      uno({
+        presets: [presetWarp()],
+      }),
```

Comment:
Is this to apply uno when we are not in shadow dom?

Reply:
Good question! That is correct. Our web components presented in the preview page have uno CSS in the scope of their shadow DOM only, so that has no effect on slotted elements that come from the outside. That's why I had to add UnoCSS to the preview page in `pages/includes/scripts.js`. Here for example I would need to have CSS for `.my-12` on the page level in order for w-card to show a correctly styled paragraph slot:
`<w-card><p class="my-12">hello</p></w-card>`

We used a similar setup in [@warp-ds/css-docs](https://github.com/warp-ds/css-docs/blob/master/docs/.vitepress/config.js#L305-L319).
Let me know if any of the above isn't clear :)